src/dappsExplorer/DAppsExplorerScreenSearchFilter.tsx

```diff
@@ -296,7 +306,7 @@ export function DAppsExplorerScreenSearchFilter() {
       {DappFavoritedToast}
       <DappRankingsBottomSheet
         forwardedRef={dappRankingsBottomSheetRef}
-        onPressDapp={onSelectDapp}
+        onPressDapp={createOnPressDappHandler}
```

Comment:
Same here. Did you mean something like this?
```suggestion
        onPressDapp={createOnPressDappHandler(...)}
```